pkgs/applications/networking/browsers/vivaldi/default.nix

```diff
@@ -85,6 +74,6 @@ in stdenv.mkDerivation rec {
     homepage    = "https://vivaldi.com";
     license     = licenses.unfree;
     maintainers = with maintainers; [ otwieracz nequissimus ];
-    platforms   = platforms.linux;
+    platforms   = with platforms; intersectLists linux x86_64;
```

Comment:
Currently both ways describes the same platforms. Using `platforms.linux` and `platforms.x86_64` looks more general to me.

Maybe we could have an attribute in `platforms` for `"x86_64-linux"`.